CorsixTH/Lua/filesystem.lua

```diff
@@ -99,8 +99,20 @@ function FileSystem:listFiles(virtual_path, ...)
   return self.files
 end
 
+local function get_full_path(virtual_path, ...)
+  if ... then
```

Comment:
lol :)

Might be useful to document the function with expected arguments, as this can be pretty much anything.

Also, more function name and function documentation stuff. Likely it exists at more spots.

Reply:
Done